app/src/main/res/values/strings.xml

```diff
@@ -518,4 +518,13 @@
     <string name="wiki_print_document_title">Wiki page %1$s in %2$s/%3$s</string>
 
     <string name="repository_owner_avatar">Repository owner avatar</string>
+    <string name="channel_notifications_name">GitHub notifications</string>
+    <string name="channel_notifications_description">Notifications received from GitHub.</string>
```

Comment:
I haven't seen the style used for other notification channel descriptions, but my gut feeling is that the trailing period should be removed.